web/src/ProviderTable.js

```diff
@@ -193,6 +193,28 @@ class ProviderTable extends React.Component {
       //     )
       //   }
       // },
+      {
+        title: i18next.t("application:rule"),
+        dataIndex: "rule",
+        key: "rule",
+        width: "100px",
+        render: (text, record, index) => {
+          if (record.provider?.category !== "Captcha") {
+            return null;
+          }
+          return (
+            <Select virtual={false} style={{width: "100%"}}
+              value={text}
+              defaultValue="None"
+              onChange={value => {
+                this.updateField(table, index, "rule", value);
+              }} >
+              <Option key="None" value="None">None</Option>
+              <Option key="Always" value="Always">Always</Option>
```

Comment:
```suggestion
              <Option key="Always" value={i18next.t("application:Always")}>Always</Option>
```

Reply:
Wrong place? @leo220yuyaodog

Reply:
Does the text in this option need to be modified?

Reply:
@hsluoyz

Reply:
> Does the text in this option need to be modified?

Yes, when switch the language, the options also need to be translated.

Reply:
> Wrong place? @leo220yuyaodog

I modified the code in wrong place.  Now it has be corrected